Tests/MSFT_xFirewall.Tests.ps1

```diff
@@ -0,0 +1,15 @@
+Remove-Module -Name MSFT_xFirewall -Force -ErrorAction SilentlyContinue
+Import-Module -Name $PSScriptRoot\..\DSCResources\MSFT_xFirewall -Force -DisableNameChecking
+
+if (! (Get-Module xDSCResourceDesigner))
+{
+    Import-Module -Name xDSCResourceDesigner -ErrorAction SilentlyContinue
+}
+
+Describe 'Schema Validation for MSFT_xFirewall' {
+    Copy-Item -Path ((get-item .).parent.FullName) -Destination $(Join-Path -Path $env:ProgramFiles -ChildPath 'WindowsPowerShell\Modules\') -Force -Recurse
+    $result = Test-xDscResource MSFT_xFirewall
```

Comment:
Never noted that before in the examples. Lots of Pester tests I've seen don't do this. Any idea why it's best practice?